test/colorchanger/colorchanger_test.exs

```diff
@@ -0,0 +1,14 @@
+defmodule Elixirlight.ColorchangerTest do
+  use ExUnit.Case, async: true
+
+  test "gets initial color" do
+    {:ok, color} = Elixirlight.Colorchanger.start_link
+    assert Elixirlight.Colorchanger.get(color) == "#"
+  end
+
+  test "set color to white" do
+    {:ok, color} = Elixirlight.Colorchanger.start_link
+    Elixirlight.Colorchanger.define_color(color, "#FFFFFF")
+    assert Elixirlight.Colorchanger.get(color) == "#FFFFF"
```

Comment:
If you're using `define_color` I would instead call this function `get_color`

Also, the thing returning from your `Colorchanger` is probably best know as something other than a `color` since it seems weird to pass in a color and to `get` or to passin both a `color` and a color value when defining your color.